starlite/contrib/sqlalchemy/repository.py

```diff
@@ -0,0 +1,540 @@
+                return instances
+
+            instances = await self.list(CollectionFilter(field_name=self.id_attribute, values=item_ids))
+            await self.session.execute(
+                delete(self.model_type).where(getattr(self.model_type, self.id_attribute).in_(item_ids))
+            )
+            await self.session.flush()
+            # no need to expunge.  The list did this already.
+            return instances
+
+    async def exists(self, **kwargs: Any) -> bool:
+        """Return true if the object specified by ``kwargs`` exists.
+
+        Args:
+            **kwargs: Identifier of the instance to be retrieved.
+
+        Returns:
+            True if the instance was found.  False if not found..
+
+        """
+        existing = await self.count(**kwargs)
+        return bool(existing > 0)
+
+    async def get(self, item_id: Any, **kwargs: Any) -> ModelT:
+        """Get instance identified by `item_id`.
+
+        Args:
+            item_id: Identifier of the instance to be retrieved.
+            **kwargs: Additional parameters
+
+        Returns:
+            The retrieved instance.
+
+        Raises:
+            RepositoryNotFoundException: If no instance found identified by `item_id`.
+        """
+        with wrap_sqlalchemy_exception():
+            statement = self._filter_select_by_kwargs(statement=self.statement, **{self.id_attribute: item_id})
+            instance = (await self._execute(statement)).scalar_one_or_none()
+            instance = self.check_not_found(instance)
+            self.session.expunge(instance)
+            return instance
+
+    async def get_one(self, **kwargs: Any) -> ModelT:
+        """Get instance identified by ``kwargs``.
+
+        Args:
+            **kwargs: Identifier of the instance to be retrieved.
+
+        Returns:
+            The retrieved instance.
+
+        Raises:
+            RepositoryNotFoundException: If no instance found identified by `item_id`.
+        """
+        with wrap_sqlalchemy_exception():
+            statement = self._filter_select_by_kwargs(statement=self.statement, **kwargs)
+            instance = (await self._execute(statement)).scalar_one_or_none()
+            instance = self.check_not_found(instance)
+            self.session.expunge(instance)
+            return instance
+
+    async def get_one_or_none(self, **kwargs: Any) -> ModelT | None:
+        """Get instance identified by ``kwargs`` or None if not found.
+
+        Args:
+            **kwargs: Identifier of the instance to be retrieved.
+
+        Returns:
+            The retrieved instance or None
+        """
+        with wrap_sqlalchemy_exception():
+            statement = self._filter_select_by_kwargs(statement=self.statement, **kwargs)
+            instance = (await self._execute(statement)).scalar_one_or_none()
+            if instance:
+                self.session.expunge(instance)
+            return instance or None
```

Comment:
Isn't instance already `ModelT | None` as the return value from `scalar_one_or_none()`, making the `or None` redundant?